image_transport/include/image_transport/subscriber.hpp

```diff
@@ -66,6 +66,7 @@ class Subscriber
   IMAGE_TRANSPORT_PUBLIC
   Subscriber() = default;
 
+  [[deprecated("Use Subscriber(rclcpp::node_interfaces...) instead")]]
```

Comment:
Deprecation message update
```suggestion
  [[deprecated("Use Subscriber(..., rclcpp::QoS, ...) instead")]]
```